### PR TITLE
v2.0.0-beta-10 add-toggle-for-user-info

### PR DESCRIPTION
**Pull request by smilburn1990**
### Added
- New prop for `userInfo` coming from outside the component
- Logic to call the data if prop is `false`

### Changed
- Name of method

### CODE REVIEW

File: packages/f-header/src/components/Header.vue [hunk 1]
```diff
@@ -19,6 +19,7 @@
                 :open-menu-text="copy.openMenuText"
                 :delivery-enquiry="copy.deliveryEnquiry"
                 :show-delivery-enquiry="showDeliveryEnquiryWithContent"
+                :user-info-prop="userInfoProp"
```
Comment (kevinrodrigues) [inline packages/f-header/src/components/Header.vue:22]: Do we need the `prop` part considering it's a `prop`? `user-info` should be sufficient.

File: packages/f-header/src/components/Navigation.vue [hunk 2]
```diff
@@ -230,13 +230,18 @@ export default {
             type: Function,
             default: () => ({}),
             required: false
+        },
+        userInfoProp: {
```
Comment (kevinrodrigues) [inline packages/f-header/src/components/Navigation.vue:234]: Similar to above.

File: packages/f-header/src/components/Navigation.vue [hunk 3]
```diff
@@ -230,13 +230,18 @@ export default {
...
+            default: false,
+            required: false
         }
     },
     data () {
         return {
             navIsOpen: false,
             currentScreenWidth: 0,
-            userInfo: false
+            userInfo: this.userInfoProp
```
Comment (kevinrodrigues) [inline packages/f-header/src/components/Navigation.vue:244]: Same as above.

File: packages/f-header/src/components/Navigation.vue [hunk 4]
```diff
@@ -275,7 +282,7 @@ export default {
         onResize () {
             this.currentScreenWidth = window.innerWidth;
         },
-        async setUserDetails () {
+        async setUserInfo () {
```
Comment (kevinrodrigues) [inline packages/f-header/src/components/Navigation.vue:285]: The catch statement would need updating from `setUserDetails` > `setUserInfo`. Plus the readme.

**Review by pikax — DISMISSED** [review]

**Review by kevinrodrigues — COMMENTED** [review]
Minor comments. You'd need the changelog and package updated too.

File: packages/f-header/src/components/Header.vue (hunk 1 above)
Comment (smilburn1990) [reply packages/f-header/src/components/Header.vue:22]: Will update changelog but cannot call in `user-info` as it already has data with the same name 👍

**Review by simonsandersje — DISMISSED** [review]

File: packages/f-header/src/components/Header.vue (hunk 1 above)
Comment (ashleynolan) [reply packages/f-header/src/components/Header.vue:22]: Where does the `user-info` prop get set then? Is it just in the `Navigation.vue` file? 

If so, you could call it the same and just pass this straight through as a prop onto that component? (I remember doing something similar in the searchbox)

I think if you pass it straight through you wouldn't need to set it as data in the `Navigation.vue` file then.

File: packages/f-header/CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v2.0.0-beta-10
+------------------------------
+*September 20, 2019*
+
+### Changed
+- Name of method
```
Comment (kevinrodrigues) [inline packages/f-header/CHANGELOG.md]: Sorry bit picky here but could we be more specific of the change.

**Review by kevinrodrigues — APPROVED** [review]

**Review by dandel10n — APPROVED** [review]

**Pull request merged**